### DISCUSSION TITLE
PowerBI API Report Pages not loading

### DISCUSSION THREAD

**Discussion by johandeconingmswa** [migueesc123/PowerBIRESTAPI#150] [General]
When I try to load the Report Pages from the Workspace Folder, it's not connecting properly when I have signed in properly.  It still gives me the error of:  
![image](https://user-images.githubusercontent.com/102005530/208801828-a89d328d-5884-4465-b56d-47fe788417b7.png)

I'm trying to build a reporting catalog but it's not allowing me to load the Report Pages, and there's no Report pages option under the Power BI Service Administrator folder/tab. Please advise how I can get the pages to load successfully.

**Comment by migueesc123** [comment]
you'll need to troubleshoot the response that you're getting from the REST API with a tool like Fiddler.

Another thing that you can try to do is to use a different tool like Postman to see if you're getting the same response from the REST API. If you are not getting the same response, then please let us know and provide some repro steps with it.

**Comment by johandeconingmswa** [comment]
Good day
Thanks for getting back to me so quickly.
I’ve used Fiddler to trace it, and you’ll notice that it becomes problematic during the loading of the specific Report Pages selection when using the API.
***@***.***
Line 42:
HTTP/1.1 502 Fiddler - DNS Lookup Failed
Date: Thu, 22 Dec 2022 05:51:41 GMT
Content-Type: text/html; charset=UTF-8
Connection: close
Cache-Control: no-cache, must-revalidate
Timestamp: 13:51:41.394

[Fiddler] DNS Lookup for "ipv6.msftconnecttest.com" failed. System.Net.Sockets.SocketException No such host is known

I then try to re-authenticate as PowerBI prompted me, and then online 51:
No Proxy-Authorization Header is present.

No Authorization Header is present.

Hope this helps you.

Johan

**Comment by migueesc123** [comment]
@johandeconingmswa are you getting the same response when using other tools / clients? a 500 error or any 5XX errors are net service level errors and not specific to the connector, so it might be something quite specific with your tenant setup.

Adding @klinejordan in case he has any suggestions

**Comment by audun-gard** [comment]
Hi. Plugin works great, except when I try adding "Report Pages" from the Navigator in Power BI Desktop. When selected, I am logged out "We couldn't authenticate with the credentials provided. Please try again" and I am not able to add Report Pages after logging in again. Any advice would be appreciated.  

![authentication_failed](https://user-images.githubusercontent.com/73523894/217250924-ce74e90c-0d68-4cb3-afb9-eef4fbd87212.PNG)
